### PR TITLE
feat(templates): Add Cosmic CMS based templates

### PR DESCRIPTION
**Pull request by pablodiehl**
Add Comic CMS based templates:

- cosmic-simple-nextjs-blog
- cosmic-simple-astro-blog
- cosmic-agency

### CODE REVIEW

File: templates/astro/cosmic-simple-astro-blog/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+yarn dev
+# or
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000/) with your browser to see the result.
+
+You can start editing the page by modifying `src/pages/index.astro`. The page auto-updates as you edit the file.
+
+## Deploy on Vercel
```
Comment (rodrigomaria) [inline templates/astro/cosmic-simple-astro-blog/README.md]: Please remove this section.

File: templates/nextjs/cosmic-agency/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+![desktop.png](https://imgix.cosmicjs.com/69313380-b156-11ee-9844-f9a09795e2a3-desktop.png)
+
+### UPDATED
+
+🛒 Now includes product management and checkout using Stripe.
+
+A website template built using Cosmic's React components, [Blocks](https://blocks.cosmicjs.com). Use this template to get familiar with Blocks and how they can be used to build common content-powered features for your websites and apps, or simply use it as a starting point for your next project. Save development time and learn content modeling best practices.
+
+## Links
```
Comment (rodrigomaria) [inline templates/nextjs/cosmic-agency/README.md:11]: Remove this section please.

File: templates/nextjs/cosmic-agency/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+4. Run the template.
+
+```
+bun dev
+```
+
+Open [http://localhost:3000](http://localhost:3000).
+
+## Deploy to Vercel
```
Comment (rodrigomaria) [inline templates/nextjs/cosmic-agency/README.md]: Remove this section please.

File: templates/nextjs/cosmic-simple-nextjs-blog/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Install all dependencies and run in development mode.
+
+```
+yarn
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000).
+
+## Deploy to Vercel
```
Comment (rodrigomaria) [inline templates/nextjs/cosmic-simple-nextjs-blog/README.md]: Remove this section please.

**Review by maxwelbm — APPROVED** [review]

**Review by rodrigomaria — APPROVED** [review]

**Pull request merged**